pom.xml

```diff
@@ -298,6 +338,31 @@
                 <artifactId>powermock-module-junit4</artifactId>
                 <version>1.6.4</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-aop</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-beans</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-context</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-core</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-expression</artifactId>
+                <version>${spring-version}</version>
+            </dependency>
```

Comment:
Doesn't the worker framework use the spring framework bom?

Reply:
Yes Dermot. The worker-framework branch SCMOD-6281 has the changes for spring upgrade. But the build is failing if we use the lastest worker framework.
New Jira created to fix build issue while upgrading latest worker-framework.

https://portal.digitalsafe.net/browse/SCMOD-7023

Reply:
OK but I want these changes reviewed and simplified as part of SCMOD-7023.  Add a comment to that effect to the description of the Jira.